From 4f40caa40df087b34e2d1f679ec1a3a2639079a4 Mon Sep 17 00:00:00 2001
From: PratikBhusal <PratikBhusal@users.noreply.github.com>
Date: Sat, 22 May 2021 22:45:06 -0500
Subject: [PATCH] Set clipboard selection to match primary selection

This patch is a port of the st 0.8.3 clipboard patch by Kai Hendry -
hendry (at) iki (dot) fi

Because it follows the "git --format-patch" file format, you may also
use "git am" to apply this patch.
---
 x.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/x.c b/x.c
index 2a3bd38..5a74804 100644
--- a/x.c
+++ b/x.c
@@ -686,6 +686,7 @@ setsel(char *str, Time t)
 	XSetSelectionOwner(xw.dpy, XA_PRIMARY, xw.win, t);
 	if (XGetSelectionOwner(xw.dpy, XA_PRIMARY) != xw.win)
 		selclear();
+	clipcopy(NULL);
 }
 
 void
-- 
2.43.0

